tempesta_fw/server.h

```diff
@@ -192,7 +192,7 @@ struct tfw_scheduler_t {
 	void			(*del_grp)(TfwSrvGroup *sg);
 	int			(*add_srv)(TfwServer *srv);
 	void			(*del_srv)(TfwServer *srv);
-	TfwSrvConn		*(*sched_grp)(TfwMsg *msg);
+	struct tfw_vhost_t	*(*sched_vhost)(TfwMsg *msg);
```

Comment:
Forward declaration for `TfwVhost` is needed in server.h, but since it is typedef - the only way to achieve this is to insert `typedef struct tfw_vhost_t TfwVhost;` into server.h, and this solution seems not appropriate.